src/main/kotlin/frc/robot/subsystems/shooter/ShooterIOReal.kt

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems.shooter
+
+import com.ctre.phoenix6.controls.VelocityVoltage
+import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.NeutralModeValue
+import edu.wpi.first.units.Angle
+import edu.wpi.first.units.MutableMeasure
+import edu.wpi.first.units.Units
+import edu.wpi.first.units.Velocity
+
+class ShooterIOReal : ShooterIO {
+    override val topRollerInputs = LoggedRollerInputs()
+    override val bottomRollerInputs = LoggedRollerInputs()
+    private val topMotor = TalonFX(-1) // TODO: Replace with real port
+    private val bottomMotor = TalonFX(-1) // TODO: Replace with real port
```

Comment:
> TODO: Replace with real port

This can be chosen right now.

Reply:
24b6a35b2d3ab73151c16a271f174f3cf62899c3